firestore/angular-rewrite/src/app/submit-review-modal/submit-review-modal.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+   * Copyright 2023 Google LLC
+   *
+   * Licensed under the Apache License, Version 2.0 (the "License");
+   * you may not use this file except in compliance with the License.
+   * You may obtain a copy of the License at
+   *
+   *      http://www.apache.org/licenses/LICENSE-2.0
+   *
+   * Unless required by applicable law or agreed to in writing, software
+   * distributed under the License is distributed on an "AS IS" BASIS,
+   * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   * See the License for the specific language governing permissions and
+   * limitations under the License.
+-->
+
+<div class="add-dialog-container">
+    <h2>Add a review!</h2>
+
+    <div id="star-rating-container">
+        <i hidden class="template material-icons star-rating" 
+        [style.color]="review.rating >= 1 ? '#feb22c' : 'gray'"
+        id="star-border-icon" (mouseenter)="review.rating = 1">star</i>
+        <i hidden class="template material-icons star-rating" 
+        [style.color]="review.rating >= 2 ? '#feb22c' : 'gray'"
+        id="star-border-icon" (mouseenter)="review.rating = 2">star</i>
+        <i hidden class="template material-icons star-rating" 
+        [style.color]="review.rating >= 3  ? '#feb22c' : 'gray'"
+        id="star-border-icon" (mouseenter)="review.rating = 3">star</i>
+        <i hidden class="template material-icons star-rating" 
+        [style.color]="review.rating >= 4 ? '#feb22c' : 'gray'"
+        id="star-border-icon" (mouseenter)="review.rating = 4">star</i>
+        <i hidden class="template material-icons star-rating" 
+        [style.color]="review.rating >= 5 ? '#feb22c' : 'gray'"
+        id="star-border-icon" (mouseenter)="review.rating = 5">star</i>
+    </div>
+    
+    <mat-form-field id="review-input-field">
+        <textarea matInput placeholder="Review Text" [(ngModel)]="review.text">
+        </textarea>
```

Comment:
nit: can self-close textarea here

Reply:
For some reason, angular isn't allowing me to self-close the text area here. I get an error that says the following when I try:
```
Only void, custom, and foreign elements can be self-closed "textarea" ngtsc(-995002)
```

It seems like native html tags can't be self closed here; Only Angular components can be.